unit_test/sparse/Test_Sparse_CrsMatrix.hpp

```diff
@@ -329,4 +368,4 @@ TEST_F( TestCategory, sparse ## _ ## crsmatrix_host_mirror ## _ ## SCALAR ## _ #
  EXECUTE_TEST(kokkos_complex_float, int64_t, size_t, TestExecSpace)
 #endif
 
-
+#undef EXECUTE_TEST
```

Comment:
@brian-kelley: Is this undef due to conflicts with gtest macros?

Reply:
@e10harvey No, that is our macro - this is so that we can include all the different test hpps into a single cpp, whereas we used to have a separate cpp for each one. The undef is just to avoid "redefinition of macro" warning, so it would never break our tests if we forget to add it.